README.md

```diff
@@ -1,20 +1,21 @@
-# liquibase-clickhouse [![maven][maven-image]][maven-url]
-                       
-[maven-image]: https://img.shields.io/maven-central/v/com.mediarithmics/liquibase-clickhouse.svg?maxAge=259200&style=for-the-badge&color=brithgreen&label=com.mediarithmics:liquibase-clickhouse
-[maven-url]: https://search.maven.org/search?q=a:liquibase-clickhouse
-Supported operations: update, rollback (with provided SQL script), tag
+# liquibase-clickhouse
```

Comment:
It would be cool to describe what the fork was made for and what new functionality was added here